### PR TITLE
Pbs argparse

### PR DESCRIPTION
**Pull request by shot0829**
#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

File: scripts/schedulers/pbs.py [hunk 1]
```diff
@@ -20,7 +20,10 @@ def get_metadata(self, db_running_task: SchedulerTask, task_logger: Logger):
         cmd = f"qstat -f -F json -x {db_running_task.job_id}"
 
         out, err = self._run_command_and_wait(cmd, shell=True)
-        json_dict = json.loads(out)
+        # special treatment after nurion added GPU replated variable in qstat that is a single \ which will cause json parser unable to read
```
Comment (jasonmotha) [inline scripts/schedulers/pbs.py:23]: replated?

Removes '\\' as nurion added a GPU ? that is a single '\' which causes json parser issues. 


In general comments should be active voice with the action at the start - I still can't figure out what replated is lol.

**Review by jasonmotha — COMMENTED** [review]
.

File: scripts/schedulers/pbs.py (hunk 1 above)
Comment (shot0829) [reply scripts/schedulers/pbs.py:23]: typo..... it was supposed to be "related" 😂

**Review by jasonmotha — COMMENTED** [review]
It feels like a few of the commands are a bit aggressively in the list rather than broken up. Do you need all the arguments to be in one argument?

**Comment by shot0829** [comment]
> It feels like a few of the commands are a bit aggressively in the list rather than broken up. Do you need all the arguments to be in one argument?

it was mostly for consistancy.

**Comment by jasonmotha** [comment]
> > It feels like a few of the commands are a bit aggressively in the list rather than broken up. Do you need all the arguments to be in one argument?
> 
> it was mostly for consistancy.

What do you mean by that?

**Comment by shot0829** [comment]
> > > It feels like a few of the commands are a bit aggressively in the list rather than broken up. Do you need all the arguments to be in one argument?
> > 
> > 
> > it was mostly for consistancy.
> 
> What do you mean by that?

so all PBS calls have the same behavior. this will reduce the chance of any pbs parsing behavior catch us off guard.
I do realize that having everything a shell call have potential risks, but with how strict pbs is on the syntax, i dont think this will be a issue.

**Review by jasonmotha — APPROVED** [review]
Doesn't quite seem like best practice. But PBS is full of that.

**Review by sungeunbae — APPROVED** [review]

**Pull request merged**